velox/expression/tests/SparkExpressionFuzzerTest.cpp

```diff
@@ -62,7 +62,11 @@ int main(int argc, char** argv) {
       "chr",
       "replace",
       "might_contain",
-      "unix_timestamp"};
+      "unix_timestamp",
+      // Skip concat_ws as it triggers a test failure due to an incorrect
+      // expression generation from fuzzer:
+      // https://github.com/facebookincubator/velox/issues/6590
```

Comment:
@PHILO-HE @unigof Let's fix the Fuzzer issue and not skip this function. What's the issue?